prow/plugins/assign/assign.go

```diff
@@ -98,6 +105,23 @@ func parseLogins(text string) []string {
 	return parts
 }
 
+func CheckMembership(gc githubClient, log *logrus.Entry, org string, members []string) error {
+	nonMembers := []string{}
+	for _, member := range members {
+		if ok, merr := gc.IsMember(org, member); merr == nil && !ok {
+			nonMembers = append(nonMembers, member)
+		} else if merr != nil {
+			log.WithError(merr).Errorf("Failed IsMember(%s, %s)", org, member)
+		}
+	}
+
+	if len(nonMembers) == 0 {
+		return nil
+	}
+
+	return fmt.Errorf("Not member of %s, %v", org, nonMembers)
```

Comment:
Can we please create something more beautiful and human than this? Not everyone working with issues is going to be a programmer.